CUDA/Gb_cuda_finalize.c

```diff
@@ -0,0 +1,8 @@
+#include "GB.h"
```

Comment:
Minor comment: the name of the file should be GB_cuda_finalize.c, not Gb_cuda_finalize.c

Reply:
Fixed